core/mainstage.go

```diff
@@ -165,17 +165,25 @@ func (st *Stage) firstWindowStages() *stages {
 
 // configMainStage does main-stage configuration steps
 func (st *Stage) configMainStage() {
+	sc := st.Scene
 	if st.NewWindow {
 		st.FullWindow = true
 	}
+	if st.NewWindow && st.Type == DialogStage && TheApp.Platform() == system.Web && st.Context != nil {
+		if st.Context.AsWidget().SizeClass() == SizeExpanded {
```

Comment:
Yes, you can make this `TheApp.Platform().IsMobile()` so that it works on native tablets/iPads etc. Another question is whether we should just require `!= SizeCompact` (to also include `SizeMedium`). Finally, this should be one if statement.

Reply:
done